doc/classes/PopupMenu.xml

```diff
@@ -503,6 +534,7 @@
 			<argument index="0" name="idx" type="int">
 			</argument>
 			<description>
+				Determines whether the checkbox or radio button of the specified [code]idx[/code] will be checked or not.
```

Comment:
Use `toggles` over determines whether

Reply:
I feel that "Determines whether" is better understood in different languages and also better translated in different languages. So you should consider adding it in the guidelines.

Reply:
> Determines whether the checkbox or radio button of the specified [code]idx[/code] will be checked or not.

The above for description for example, perfectly translates to my native language. You could use these to also automatically translate to other languages also?

Reply:
These are English docs, the goal is to write clear English. Then, every language is different and translation work is different for each of them. For instance, does not translate well to French, my native language, or the other languages I speak (Spanish, Portuguese, Japanese).